packages/hooks/src/__tests__/useRegisterPasskey.test.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { renderHook, act } from '@testing-library/react-hooks';
+import APIProvider from '@deriv/api/src/APIProvider';
+import { WS } from '@deriv/shared';
+import useRegisterPasskey from '../useRegisterPasskey';
+import { startRegistration } from '@simplewebauthn/browser';
+
+jest.mock('@simplewebauthn/browser', () => ({
```

Comment:
@yauheni-deriv can we improve it?  Again, for your consideration, no pressure 🧐
<img width="1037" alt="Screenshot 2024-03-04 at 16 10 47" src="https://github.com/binary-com/deriv-app/assets/103181650/71a54749-fc5c-4835-b75d-47117c34f6d7">

Reply:
will improve it to 100% 👍